Makefile

```
@@ -0,0 +1,50 @@
+include .env
+ifdef ENV
+include .env.$(ENV)
+endif
+
+.PHONY: env-setup env-teardown db/migrate db/rollback migration/create migration/status dev install-dependencies test wait-for-postgres
+
+env-setup:
+	docker-compose -f docker-compose.dev.yml up -d
+
+env-teardown:
+	docker-compose -f docker-compose.dev.yml down
+
+db/migrate:
+	make wait-for-postgres
+	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" up
+
+db/rollback:
+	make wait-for-postgres
+	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" down
+
+migration/create:
+ifndef MIGRATION_NAME
+	$(error MIGRATION_NAME is required)
+endif
+	goose -dir database/migrations create $(MIGRATION_NAME) sql
+
+migration/status:
+	goose -dir database/migrations -table "migration_versions" postgres "$(DATABASE_URL)" status
+
+dev:
+	make env-setup
+	make db/migrate
+	forego start -f Procfile.dev
+
+install-dependencies:
+	go install github.com/cosmtrek/air@v1.41.1
```

```suggestion
	go install github.com/cosmtrek/air@v1.44.0
```

I had an error message telling me it cannot find the v1.41.1 🤔